backend/api/v1/endpoints/research.py

```
@@ -9,14 +9,15 @@
 from models.user import User
 from models.research_collaboration import ResearchCollaboration
 from models.collaboration_request import CollaborationRequest
+from typing import List
 from fastapi import Query
 from sqlalchemy import and_, func, or_
 from models.research_collaboration import research_collaborators
 from werkzeug.utils import secure_filename
 import uuid
```

Ruff (0.8.2)

`uuid` imported but unused

Remove unused import: `uuid`

(F401)